Recipe informations

rafamanzo opened the pull request:
Here we have two fixes:
- `MezuroInformation` widespread usage (specially recipes): this ensures that we have informations like version in only one file making easy releases
- Package upload and publish: the upload has to create the file with its extension (.deb or .rpm) in order to get the metada properly generated

rafamanzo committed:
Pull informations on recipes

This way we centralize all the versions and relative information into
one file to rule them all.

rafamanzo committed:
Fix MezuroInformations retrieval for recipes

The mezuro_informations.rb was not accessible inside containers and
there was missing requires.

Signed off by: Eduardo Araújo <duduktamg@hotmail.com>

rafamanzo committed:
Fix file path retrieval

It was giving the wrong MezuroInformations hash entry to the path
helper.

Signed off by: Eduardo Araújo <duduktamg@hotmail.com>

rafamanzo committed:
Update repository metadata

Without metadata updates the clients never get notice of existing or
updated packages.

marcheing committed:
Fix package publishing

The publishing should get done by the upload request using the parameter
publish=1 on the request url.

Signed off by: Eduardo Araújo <duduktamg@hotmail.com>
Signed off by: Rafael Reggiani Manzo <rr.manzo@gmail.com>
Signed off by: Diego Araújo <diegoamc90@gmail.com>

Commit:
Fix rpm publishing

Before this, the metadata wasn't being generated.

Signed off by: Rafael Reggiani Manzo <rr.manzo@gmail.com>

diegoamc commented:
Seems good to me! Good job, guys!

@marcheing and @danielkza , please review this PR.

marcheing committed:
Change ` to ' on post_install script cat input file

It was actually executing a command instead of just interpolating the
string. This behaviour generated an empty string after 'please run'

Signed off by: Eduardo Araújo <duduktamg@hotmail.com>

rafamanzo committed:
Fix admin scripts installation

Previously the admin scripts were getting installed on a weird path not
accessible through the system path. This was leading to errors on the
postinstall script.

Signed off by: Eduardo Araújo <duduktamg@hotmail.com>

rafamanzo committed:
Bump package release

The latest commits fixed issues with the packaging postinstall. So we
bump the package release version.

Signed off by: Eduardo Araújo <duduktamg@hotmail.com>

rafamanzo commented:
Don't merge before: https://github.com/mezuro/packaging/pull/9

rafamanzo committed:
Use release number from MezuroInformations

The recipes were harcoded with value 1.

diegoamc committed:
Merge pull request #9 from mezuro/fix_package_post_install

Change ` to ' on post_install script cat input file
